tests/Admin/PoolTest.php

```diff
@@ -285,6 +263,25 @@ public function getNonStringAdminServiceNames(): array
         ];
     }
 
+    public function testGetAdminByAdminCodeWithCodeNotChild(): void
```

Comment:
This test is not new, it seems like we changed the order on this class, is the same on 3.x?

Reply:
I'l try to move it, the think is that in 3.x, this test below the `getNonStringAdminServiceNames` method.

https://github.com/sonata-project/SonataAdminBundle/blob/c30e1ce0567b803490c7b647a6c62d0553e20b31/tests/Admin/PoolTest.php#L281

and in `master` below `testGetAdminByAdminCodeWithInvalidCode`:

https://github.com/sonata-project/SonataAdminBundle/blob/ff34fc6c04e79a6f5fb01a829ade4ce93e1938cf/tests/Admin/PoolTest.php#L246

and I picked the one from `3.x` which seemed more updated to me.

Reply:
No problem then